src/components/Layout/PreventScrollContainer.tsx

```diff
@@ -0,0 +1,7 @@
+import { PropsWithChildren } from "react";
+import { Css } from "src/Css";
+
+/** Intended to wrap the whole application to prevent the brower's native scrolling behavior while also taking the full height of the viewport */
+export function PreventScrollContainer({ children }: PropsWithChildren<{}>) {
```

Comment:
Kinda wonder if this could be inside of `NestedScrollLayoutContextProvider`? Like wouldn't they be used in pairs where a top-level `PreventScrollContainer` would have an immediate child of `NestedScrollLayoutContextProvider`?

Or maybe not, looking at the stories, still reading...

Reply:
That is what I was thinking too at first. Though, because I wanted to use `NestedScrollLayoutContextProvider` twice, once in the SideNav, and another within the page's content. Because of this, I figured it'd be better not to wrap the whole application, but rather wrap it further down into the page where we know the scrolling will take place.
With that said, this was just a first go. I'm happy to explore other approaches.